backend/src/main/java/com/capstone/moneytree/service/impl/DefaultUserService.java

```diff
@@ -70,217 +70,210 @@
     private final MoneyTreePasswordEncryption passwordEncryption;
     private static final Logger LOG = LoggerFactory.getLogger(DefaultUserService.class);
     private final AmazonS3Service amazonS3Service;
-    
+
     private final String bucketName;
-    
+
     @Autowired
-    public DefaultUserService(UserDao userDao, FollowsDao followsDao, ValidatorFactory validatorFactory,
-            AmazonS3Service amazonS3Service, @Value("${aws.profile.pictures.bucket}") String bucketName) {
+    public DefaultUserService(UserDao userDao, FollowsDao followsDao, ValidatorFactory validatorFactory, AmazonS3Service amazonS3Service,
+                              @Value("${aws.profile.pictures.bucket}") String bucketName) {
         this.userDao = userDao;
         this.followsDao = followsDao;
         this.validatorFactory = validatorFactory;
         this.passwordEncryption = new MoneyTreePasswordEncryption();
         this.amazonS3Service = amazonS3Service;
         this.bucketName = bucketName;
     }
-    
+
     @Override
     public Iterable<User> getAllUsers() {
-       return userDao.findAll();
+        return userDao.findAll();
     }
-    
+
     @Override
     public User getUserById(Long id) {
-       User user = userDao.findUserById(id);
-       if (user == null) {
-          throw new EntityNotFoundException(USER_NOT_FOUND);
-       }
-       return user;
+        User user = userDao.findUserById(id);
+        if (user == null) {
+            throw new EntityNotFoundException(USER_NOT_FOUND);
+        }
+        return user;
     }
-    
+
     @Override
     public User getUserByEmailAndUsername(String email, String username) {
-       User existingUser = userDao.findUserByEmailAndUsername(email, username);
-       if (existingUser == null) {
-          throw new EntityNotFoundException(USER_NOT_FOUND);
-       }
-       LOG.info("Found user with email {}", existingUser.getEmail());
-       return existingUser;
+        User existingUser = userDao.findUserByEmailAndUsername(email, username);
+        if (existingUser == null) {
+            throw new EntityNotFoundException(USER_NOT_FOUND);
+        }
+        LOG.info("Found user with email {}", existingUser.getEmail());
+        return existingUser;
     }
-    
+
     @Override
     public User createUser(User user) {
-       getUserValidator().validate(user);
-       
-       String password = user.getPassword();
-       String encryptedPassword = encryptData(password);
-       user.setPassword(encryptedPassword);
-       user.setAvatarURL(String.format("https://%s.s3.amazonaws.com/%s", bucketName, DEFAULT_PROFILE_NAME));
-       
-       userDao.save(user);
-       
-       LOG.info("Created user: {}", user.getUsername());
-       
-       return user;
+        getUserValidator().validate(user);
```

Comment:
I've seen multiple "intend-only" changes difference. Please make sure you don`t push those next time as the PR becomes quite big to review.

Reply:
oh man, I used VSCode for my first commit, and then IntelliJ for other commits. I tried to reformat backend files using IntelliJ but it added some new indentations, I am trying to find a way to revert it.

Reply:
Yeah use one or the other. Use IntelliJ for backend and then switch to VSCode for the frontend.